src/main/java/servlets/BillingConfig.java

```diff
@@ -66,7 +66,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
      * TODO: @cade, eventually you will want to pass this access token into the 
      * updateSheets() method in the SheetsConverter class.
      */
-    HttpSession session = request.getSession();
+    HttpSession session = request.getSession(false);
```

Comment:
please document what this parameter means

Reply:
getSession(false) will only get a session if one exists. getSession(true) will create a session if none currently exist.